CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - Added sensuctl commands to manage the `APIKey` resource.
 - Added support for api keys to be used in api authentication.
```

Comment:
I'm not sure how/why we have API keys under the Unreleased header... I don't think that should be here at all.

Reply:
This change will eventually get merged into master, no? Was hoping to avoid more manual changelog resolutions, but happy to remove this if it doesn't make sense.

Reply:
IMO it is harmless to have stuff in the unreleased header, even if it doesn't exist in the branch.